src/components/order-service/order-service.controller.ts

```diff
@@ -62,6 +62,8 @@ export class OrderServiceController implements angular.IController {
     this.ctrl.serviceName = this.ctrl.serviceClass.name;
     this.ctrl.description = this.ctrl.serviceClass.description;
     this.ctrl.longDescription = this.ctrl.serviceClass.longDescription;
+    this.ctrl.docUrl = this.ctrl.serviceClass.resource.spec.externalMetadata.documentationUrl;
```

Comment:
I'm not sure `externalMetadata` is required. This might be a runtime error. Let's change it to

```js
this.ctrl.docUrl = _.get(this.ctrl.serviceClass, 'resource.spec.externalMetadata.documentationUrl');
this.ctrl.supportUrl = _.get(this.ctrl.serviceClass, 'resource.spec.externalMetadata. supportUrl');
```